tests/VM_with_planetary_network/planetary_network_test.go

```diff
@@ -1,6 +1,3 @@
-//go:build integration
```

Comment:
shouldn't it have integration build tag?

Reply:
why? I removed it because it ruins the build of the file, and not sure what is the point of having this?

Reply:
it just separates integration tests from other unit tests so it won't run unless typing the integration tag explicitly and also because it may take so long to finish so you might not want to run it with other unit tests. as for ruining the build of the file i am not familiar with that the tests run normally by executing `go test -tags=integration`